torch/_dynamo/variables/builtin.py

```diff
@@ -358,7 +358,39 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
-    def _call_min_max(self, tx, a, b):
+    def _call_min_max(self, tx, *args):
+        if all(isinstance(a, variables.ConstantVariable) for a in args):
```

Comment:
Is this case actually needed?  I'd expect just the two arg version to work fine with the old code.